src/app/common/components/MevRelaysBadge/MevRelaysBadge.tsx

```diff
@@ -29,9 +29,9 @@ const mevIcons: Record<string, string> = {
     [MEVS.ULTRA_SOUND]: 'ultra_sound',
 };
 
-const MevRelaysBadge = ({ label }: { label: string }) => {
+const MevRelaysBadge = ({ label, darkMode }: { label: string, darkMode: boolean }) => {
```

Comment:
Dark mode can be used here itself instead of passing it as property.
No sense to pass it as property because this is something not functionality-related and brings unnecessary complication to the component

Reply:
it is better to keep leaf component unaware of outside world - not to connect it to store. this makes the component more reusable and less coupled with specific implementation

Reply:
V